app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -273,13 +273,16 @@ def create_vm(args = {})
       template = template(args[:template]) if args[:template]
       instance_type = instance_type(args[:instance_type]) if args[:instance_type]
 
+      args[:cluster] = verify_attribute(self.clusters, args[:cluster])
+      cluster_networks = self.networks(:cluster_id => args[:cluster])
+
```

Comment:
can't this be inside create_interfaces function?

Reply:
The cluster-id is needed to get the relevant networks, so it is possible to send it to create_interfaces function as a parameter and then find the networks inside create_interfaces. Do you think its a better way?

Reply:
yes, I think that's better. thanks :)